DFC.App.Account.Services.DSS/Services/DssService.cs

```diff
@@ -45,14 +47,11 @@ public async Task<Customer> CreateCustomerData(Customer customerData)
             if(customerData == null)
                 return null;
 
-            var postData = new StringContent(
-                JsonConvert.SerializeObject(customerData),
-                Encoding.UTF8,
-                MediaTypeNames.Application.Json);
-            postData = AddDefaultRequestHeaders(postData);
-            postData.Headers.Add("version", _dssSettings.Value.CustomerApiVersion);
+            
+            var request = await BuidRequestMessage();
```

Comment:
Customer data is no longer assigned to request, and BuildRequestMessage has a typo

Reply:
Fixed